scripts/lunch.coffee

```diff
@@ -0,0 +1,66 @@
+# Description:
+#   Lunch recording technology.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   <botname> lunch at [location]
+#   <botname> lunch me
+
+makeRecord = (now = new Date()) ->
+
+  return {
+    last_lunch: now.getFullYear() + "-" + (now.getMonth() + 1) + "-" + now.getDate()
+    timestamp: now.getTime()
+  }
+
+module.exports = (robot) ->
+
+  robot.respond /lunch at (.+)$/i, (msg) ->
+
+    location = msg.match[1]
+    robot.brain.data.lunch_spots ||= {}
+
+    if record = robot.brain.data.lunch_spots[location]
+      msg.send "last lunch at #{location} was #{record.last_lunch}"
+    else
+      msg.send "have a great lunch!"
+
+    robot.brain.data.lunch_spots[location] = makeRecord()
+
+
+  robot.respond /lunch seed$/i, (msg) ->
```

Comment:
While good for testing, I don't think we should check this in; too easy to accidentally wipe out the valuable lunch database...! :smile: